src/07-even-more-loops.js

```diff
@@ -126,7 +168,9 @@ function findLargest(array) {}
  *  findEqual([ 0, 11, -2, 5 ], 9);
  *  //> false
  */
-function findEqual(array, selected) {}
+function findEqual(array, selected) {
+  return array.includes(selected);
```

Comment:
how would we write this without includes?  It's always a good idea to see if we can code up our own version of native methods.

Reply:
Funnily enough I spent longer than I want to admit writing my own alphabetical sort for a lvl 8 codewars problem.  Maybe a day later I found out about .sort(). Some expletives may have been said when I made that discovery.